providers/os/connection/snapshot/volumemounter.go

```diff
@@ -53,18 +53,36 @@ func (m *VolumeMounter) Mount() error {
 	return m.mountVolume(fsInfo)
 }
 
-func (m *VolumeMounter) createScanDir() error {
+// Mounts a specific partition and returns the directory it was mounted to
+func (m *VolumeMounter) MountP(partition *PartitionInfo) (string, error) {
```

Comment:
this no longer runs `lsblk` again, it simply tries to mount what we're telling it to